### PR TITLE
:construction_worker: add symfony thanks to (not) allowed plugins (1.11)

### PR DESCRIPTION
**Pull request by Nek-**
Discussion
----------

| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.11                  |
| Bug fix?        | yes                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets |                       |
| License         | MIT                                                          |

This is https://github.com/Sylius/Sylius/pull/14582 but for 1.11 branch

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -228,7 +228,8 @@
         "sort-packages": true,
         "allow-plugins": {
             "symfony/flex": true,
-            "dealerdirect/phpcodesniffer-composer-installer": false
+            "dealerdirect/phpcodesniffer-composer-installer": false,
+            "symfony/thanks": true
```
Comment (loic425) [inline composer.json]:
```suggestion
            "symfony/thanks": false
```

Comment (Nek-) [reply composer.json]: Aw yes sorry

**Review by loic425 — APPROVED** [review]

**Comment by loic425** [comment]
@Nek- Great, this is perfect, we just have to fix 1.11 branch CI in another PR.

**Comment by Nek-** [comment]
Not sure what you mean here but just in case there's also another PR for 1.12+ here #14582

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
I approved this PR, but later did I check that [we had already removed it](https://github.com/Sylius/Sylius/commit/69c663edeaefc42c2561d29abefaa9f729482d30) and this dependency is not required in 1.11. Why is this change needed?

**Comment by Nek-** [comment]
@GSadee it's about symfony/thanks not dealerdirect/phpcodesniffer-composer-installer. And it's required because it's blocking installation for developer. (I basically clone the repository and then ran composer install)

**Comment by GSadee** [comment]
> @GSadee it's about symfony/thanks not dealerdirect/phpcodesniffer-composer-installer. And it's required because it's blocking installation for developer. (I basically clone the repository and then ran composer install)

Yes, I see that it is about `symfony/thanks` and we've removed that dependency in [the mentioned commit](https://github.com/Sylius/Sylius/commit/69c663edeaefc42c2561d29abefaa9f729482d30#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34L108), so since 1.11, it seems not to be needed. 

To be 100% sure, I've checked the composer installation on the 1.11 branch and it looks like there is no problem anymore as Sylius itself does not install this dependency. Probably it was needed on the 1.10 branch 🤔.

According to that, I'm closing this PR. Of course, feel free to comment or reopen if I have missed something 🖖🏻.

**Comment by Nek-** [comment]
I must have run it on old code. Idk, sorry for the noise 🤔

**Pull request closed without merging**